x-pack/plugins/security_solution/cypress/integration/timelines/data_providers.spec.ts

```diff
@@ -46,4 +65,29 @@ describe('timeline data providers', () => {
       done();
     });
   });
+
+  it('persists timeline when data provider is updated by dragging a field from data grid', () => {
+    updateDataProviderbyDraggingField('host.name', 0);
+    waitForTimelineChanges();
+    cy.wait(1000);
+    cy.reload();
+    cy.get(`${GET_TIMELINE_GRID_CELL('host.name')}`)
+      .first()
+      .then((hostname) => {
+        cy.get(TIMELINE_DATA_PROVIDERS_CONTAINER).contains(`host.name: "${hostname.text()}"`);
+      });
+  });
+
+  it('presists timeline when a field is added by hover action "Add To Timeline" in data provider ', () => {
+    updateDataProviderByFieldHoverAction('host.name', 0);
+    cy.get(HOVER_ACTIONS.ADD_TO_TIMELINE).should('be.visible');
+    cy.get(HOVER_ACTIONS.ADD_TO_TIMELINE).trigger('click', { force: true });
```

Comment:
do you need a `cy.reload()`here as well to test the persistence?

Reply:
@michaelolo24 , great catch. 🤦 I will add it now.